pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -18,6 +18,7 @@ social:
   "og:video:width": ""
   "og:video:height": ""
   "og:video:type": ""
+video_player: true
```

Comment:
Oops it already had it at the top, just didn't see it because I usually put it last.